ci-operator/config/stolostron/policy-collection/stolostron-policy-collection-main__opp-ocp4.12-interop.yaml

```diff
@@ -50,6 +50,7 @@ tests:
     - ref: ipi-install-monitoringpvc
     - chain: ipi-install
     test:
+    - chain: cucushift-installer-check-cluster-health
```

Comment:
Can you move this line after the line:
```
- ref: acm-policies-openshift-plus
```

That way it runs the test with OPP already installed.  Thanks